_shows/the-topdollar-hour.md

```diff
@@ -0,0 +1,10 @@
+---
+show-id: the-topdollar-hour
+name: The TopDollar Hour
+hosts: [ tunacan ]
+image: the-topdollar-hour.jpg
+tags: [ rock ]
+tagline: Musical Enjoyment for the Bipolar!
+---
+
+A delightful mix of power pop, garage rock, hillbilly,  no wave, noise rock, twang and plenty of disinformation that will soothe the left elbow.
```

Comment:
Here I add the show file in the `_shows` directory. It begins with a meta section, denoted by the three dashes (a FrontMatter format used with Jekyll and other static site generators).

`show-id:` is used to link to (for the schedule page or elsewhere)
`name:` is the text used in headings or elsewhere it is needed
`hosts: [ tunacan ]` this is the hosts `array` denoted by the brackets. If more than one host, separate by comma. The host should should be reference by the filename of their corresponding people page.
image:` the code knows where to look so we only use the filename and no path
`tags: [ rock ]` another comma separated array
`tagline:` This is used on the little schedule on the home page but could work elsewhere too

And then the text after the meta area is just text. I believe the shows page reads all of the show files and uses the data and the body (outside of the three dashes) to build the shows page.